package.json

```diff
@@ -7,6 +7,7 @@
   "repository": "Polymer/lit-html",
   "main": "lit-html.js",
   "module": "lit-html.js",
+  "typings": "lit-html.d.ts",
```

Comment:
Is this necessary?

Reply:
No, it is not! As I add `husky` and `lint-staged` to the bottom, I just move the `typings` near the `main` and `module` fields. Do you prefer to keep it to the bottom?

Reply:
Nope, I just missed that this was a move of an existing field